README.md

```diff
@@ -43,54 +42,27 @@ Contents:
 
 <!-- TOC -->
 
+- [How to build Valora](#build)
 - [Celo's Mission - Prosperity for All](#mission)
-- [The Celo Stack](#stack)
+- [The Valora Stack](#stack)
 - [Documentation](#docs)
 - [Issues](#issues)
 - [Repo Structure](#repo)
 - [Contributing](#contributing)
 - [Ask Questions, Find Answers, Get in Touch](#ask)
 - [License](#license)
   <!-- /TOC -->
+  
+## 🥅 <a id="build"></a>How to Build Valora Locally 
+Follow instructions [here](https://github.com/celo-org/celo-monorepo/blob/master/SETUP.md#running-the-mobile-wallet). (This part of the READ.me is in construction! To be rewritten) 
```

Comment:
I see you're using the term READ.me here and in this PR. Do you mean README.md? I think that's the more common name for these files.

Reply:
oops! yes haha